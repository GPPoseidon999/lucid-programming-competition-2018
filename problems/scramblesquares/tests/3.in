3 3
7 2 0 7
4 7 7 7
9 2 9 1
5 6 8 5
9 5 3 7
0 7 6 3
0 2 0 6
6 7 5 2
5 3 9 5
