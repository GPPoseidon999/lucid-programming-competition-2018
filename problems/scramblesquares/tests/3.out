\ 5 / \ 0 / \ 4 /
5 X 3 3 X 7 7 X 7
/ 9 \ / 6 \ / 7 \
\ 9 / \ 6 / \ 7 /
1 X 2 2 X 7 7 X 2
/ 9 \ / 5 \ / 0 \
\ 9 / \ 5 / \ 0 /
7 X 5 5 X 6 6 X 2
/ 3 \ / 8 \ / 0 \
